test/data/parser/failure/actual/missingClosingBracket.txt

```diff
@@ -0,0 +1 @@
+RenderedError {_code = ParseErrorCode, _message = "ParseError: Unexpected token '}'.", _span = Span {start = AlexSourcePos {line = 0, col = 12}, end = AlexSourcePos {line = 1, col = 1}}}
```

Comment:
It might be cool in future if we can find a way to render the errors without reference to the implementation - I.e. describe the error and the position, but not reference Alex, etc.

Reply:
Not a blocker.

Reply:
I agree. I think that would fall under issue #7